django_elasticsearch_dsl/fields.py

```diff
@@ -1,7 +1,7 @@
 import collections
 from types import MethodType
-import warnings
 
+from elasticsearch_dsl.field import Keyword, Text
```

Comment:
These imports should be together with the ones in line 11.

Reply:
Thanks!